app/assets/javascripts/reports/reports_datatable.js

```diff
@@ -6,6 +6,7 @@
   const RETRY_COUNT = 25;
   const START_POLLING_INTERVAL = 10000;
   var CHECKED_REPORTS = [];
+  var REPORTS_TABLE;
```

Comment:
I'm not sure what is the correct naming convention for object variables, but using capitals for this doesn't seem right.